File: kafka3/src/main/scala/com/stulsoft/kafka3/AutoCommit.scala
/*
 * Copyright (c) 2017. Yuriy Stul
 */

package com.stulsoft.kafka3


/**
  * @author Yuriy Stul
  */
object AutoCommit extends Enumeration {
  type AutoCommit = Value
  val EnabledAutoCommit: Value = Value("true")
  val DisabledAutoCommit: Value = Value("false")
}

File: pkafka/kafkaj/src/main/java/com/stulsoft/kafkaj/AutoCommit.java
package com.stulsoft.kafkaj;

/**
 * @author Yuriy Stul.
 */
public enum AutoCommit {
    EnabledAutoCommit(true),
    DisabledAutoCommit(false);
    private final boolean value;
    AutoCommit(final boolean value){
        this.value=value;
    }

    @Override
    public String toString() {
        return value? "true":"false";
    }
}
